packages/web/app/views/patients/panes/EncounterMedicationPane.jsx

```diff
@@ -98,6 +104,7 @@ export const EncounterMedicationPane = React.memo(({ encounter, readonly }) => {
             disabled={readonly}
             variant="outlined"
             color="primary"
+            onClick={onMarClick}
```

Comment:
```suggestion
            onClick={navigateToMar}
```
You could simplify this by passing navigateToMar to onClick